benchmarks/tosi_et_al_2015_gcubed/tosi.cc

```diff
@@ -0,0 +1,621 @@
+#include <aspect/geometry_model/interface.h>
```

Comment:
don't we have a common header with a copyright note here for all of the cookbooks?

Reply:
The benchmarks don't seem to have a copyright statement.

Reply:
I see. That's probably something we ought to fix, but there's no need to do it here.

Reply:
See #2416.